src/Mod/Sketcher/Gui/CommandSketcherBSpline.cpp

```diff
@@ -440,24 +440,35 @@ void CmdSketcherConvertToNURB::activated(int iMsg)
     const std::vector<std::string> &SubNames = selection[0].getSubNames();
     Sketcher::SketchObject* Obj = static_cast<Sketcher::SketchObject*>(selection[0].getObject());
 
+    // Without this line there are some segfault-like issues on some dragged selections.
+    // See https://forum.freecadweb.org/viewtopic.php?f=19&p=573388#p573388.
+    // Possibly some vertices that are deleted after the conversion remain in the selection.
+    getSelection().clearSelection();
+
     bool nurbsized = false;
 
     openCommand(QT_TRANSLATE_NOOP("Command", "Convert to NURBS"));
 
-    for (size_t i=0; i < SubNames.size(); i++) {
+    std::vector<int> GeoIdList;
+
+    for (auto subName : SubNames) {
```

Comment:
Should be `const auto&` to avoid to create a tmp. copy.

Reply:
Done.